pkg/workflows/wasm/sdk.go

```diff
@@ -1,7 +1,12 @@
 package wasm
 
-import "github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
+import (
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
+	"go.uber.org/zap"
+)
 
-type Runtime struct{}
+type Runtime struct {
+	Logger *zap.Logger
```

Comment:
@agparadiso Can we make this a logger.Logger too if that's what we use as standard?